### ISSUE TITLE
Wrong error context for gp_dump_query_oids()

### ISSUE THREAD

**Issue by hlinnaka** [greenplum-db/gpdb#854]
```
postgres=# select gp_dump_query_oids('select * from invalid');
ERROR:  relation "invalid" does not exist
LINE 1: select gp_dump_query_oids('select * from invalid');
                      ^
```

The arrow pointing to the location of the error is wrong.

**Comment by danielgustafsson** [comment]
It seems to point to where `invalid` would be if the query passed to the function was the query itself being executed (which given the function makes sense since that's what actually happens).

**Comment by hlinnaka** [comment]
Fixed by PR #1629.
